scripts/globals/weaponskills.lua

```diff
@@ -959,3 +959,61 @@ function takeWeaponskillDamage(defender, attacker, params, finaldmg, slot, tpHit
 
     return finaldmg;
 end
+
+function applyAftermathEffect(player, tp, spower_1, spower_2, spower_3)
+    spower_1 = spower_1 or 1
+    spower_2 = spower_2 or 1
+    spower_3 = spower_3 or 1
+
+    -- AFTERMATH LEVEL 1
+
+    if ((tp >= 1000) and (tp <= 1100)) then
```

Comment:
`utils.clamp()` might be applicable there. But if what you have works its not really a problem (pretty sure that works as you have it, I'm just not used to seeing it done like that),